### ISSUE TITLE
milliseconds not correctly showing in utils.dateAxisLabelFormatter ? 

### ISSUE THREAD

**Issue by eddified** [danvk/dygraphs#695]
I don't have a repro for this, because I'm reporting what I **strongly** believe to be a bug, but I've never actually seen the bug exhibited. I merely found it by looking at the code.

In dygraph-utils.js, we see:
```javascript
  var year = accessors.getFullYear(date),
      month = accessors.getMonth(date),
      day = accessors.getDate(date),
      hours = accessors.getHours(date),
      mins = accessors.getMinutes(date),
      secs = accessors.getSeconds(date),
      millis = accessors.getSeconds(date);
```

Shouldn't that last line be this instead?
```javascript
      millis = accessors.getMilliseconds(date);
```
(was calling `getSeconds` when it should be calling `getMilliseconds`)
Please take a look. Thanks.

**Comment by TonyDIRL** [comment]
I'd also like to know if this is a bug or intentional.
Thanks

**Comment by danvk** [comment]
Looks like a bug. I'd be very interested to see a demonstration of it.

**Comment by eddified** [comment]
I've never used millisecond nor second resolution with Dygraphs. (For me, seconds are always 0, and milliseconds are also always 0. So swapping them won't change anything.) So if it is a bug, it certainly won't affect me. :)

**Comment by TonyDIRL** [comment]
Hi,

Would this bug cause the problems discussed in issue https://github.com/danvk/dygraphs/issues/654?

Thanks

**Comment by danvk** [comment]
Closed by #774.
